src/engine/Service.h

```diff
@@ -102,6 +102,9 @@ class Service : public Operation {
   static TripleComponent bindingToTripleComponent(
       const nlohmann::json& binding);
 
+  static std::optional<std::string> idToValueForValuesClause(
```

Comment:
Add a short comment here, especially what the `optional` means (consult my other comments first).